sonic-net/sonic-utilities
# [config] Added/updated vlan commands.

1) list add/del support for vlan.
2) Enhanced "config vlan add" command to take optional param "type" with values "management/control".  The same added in list support as well. Purpose is to identify the vlan type.
   This is optional configuration. If not set, it will treated as regular vlan (current behavior).
config_db.json:
===============
    "Vlan215": {
        "vlanid": "215"
    },
    "Vlan101": {
        "vlan_type": "control",
        "vlanid": "101"
    },
    "Vlan2156": {
        "members": [
            "Ethenret0",
        "Ethenret1"
        ],
        "vlan_type": "management",
        "vlanid": "2156"
    }

Signed-off-by: Rajkumar Pennadam Ramamoorthy <rpennadamram@marvell.com>



**- What I did**
1) list add/del support for vlan.
2) Enhanced "config vlan add" command to take optional param "type" with values "management/control".   The same added in list support as well. Purpose is to identify the vlan type.
   This is optional configuration. If not set, it will treated as regular vlan (current behavior).
**- How I did it**
config vlan add 215
config vlan add 2156 -t mgmt
config vlan add 2156 --type ctrl
config vlan add 2157 --type ctrl
config vlan list add 1500 1501 101 --type ctrl
config vlan list add -t mgmt 1510
config vlan list del 215 1510 1500

**- How to verify it**
Check the out in show running config and verify the "vlan_type" 
"Vlan215": {
        "vlanid": "215"
    },
    "Vlan101": {
        "vlan_type": "control",
        "vlanid": "101"
    },
    "Vlan2156": {
        "members": [
            "Ethenret0",
        "Ethenret1"
        ],
        "vlan_type": "management",
        "vlanid": "2156"
    }
**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**
